<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>man_hinh_them_cau_hoi</class>
 <widget class="QDialog" name="man_hinh_them_cau_hoi">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>960</width>
    <height>540</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <layout class="QGridLayout" name="gridLayout" columnstretch="1,3">
   <item row="0" column="0">
    <widget class="QGroupBox" name="groupBox">
     <property name="styleSheet">
      <string notr="true">border: 3px solid rgb(0, 112, 192);
border-radius: 10px;
background-color: rgb(255, 255, 255);</string>
     </property>
     <property name="title">
      <string/>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout" stretch="1,1,2">
      <property name="topMargin">
       <number>50</number>
      </property>
      <property name="bottomMargin">
       <number>30</number>
      </property>
      <item>
       <widget class="QGroupBox" name="groupBox_3">
        <property name="styleSheet">
         <string notr="true">border: None;</string>
        </property>
        <property name="title">
         <string/>
        </property>
        <widget class="QLabel" name="label">
         <property name="geometry">
          <rect>
           <x>0</x>
           <y>50</y>
           <width>71</width>
           <height>21</height>
          </rect>
         </property>
         <property name="font">
          <font>
           <family>Tahoma</family>
           <pointsize>12</pointsize>
           <bold>true</bold>
          </font>
         </property>
         <property name="text">
          <string>Độ khó</string>
         </property>
        </widget>
        <widget class="QComboBox" name="do_kho">
         <property name="geometry">
          <rect>
           <x>70</x>
           <y>45</y>
           <width>111</width>
           <height>31</height>
          </rect>
         </property>
         <property name="styleSheet">
          <string notr="true">border: 2px solid rgba(46, 82, 101, 200);
</string>
         </property>
        </widget>
       </widget>
      </item>
      <item>
       <widget class="QGroupBox" name="groupBox_4">
        <property name="styleSheet">
         <string notr="true">border: None;</string>
        </property>
        <property name="title">
         <string/>
        </property>
        <widget class="QLineEdit" name="diem">
         <property name="geometry">
          <rect>
           <x>70</x>
           <y>26</y>
           <width>111</width>
           <height>31</height>
          </rect>
         </property>
         <property name="styleSheet">
          <string notr="true">border: 2px solid rgba(46, 82, 101, 200);
border-radius: 10px;
padding: 0 8px;
selection-background-color: darkgray;
font-size: 16px;</string>
         </property>
        </widget>
        <widget class="QLabel" name="label_2">
         <property name="geometry">
          <rect>
           <x>0</x>
           <y>30</y>
           <width>51</width>
           <height>21</height>
          </rect>
         </property>
         <property name="font">
          <font>
           <family>Tahoma</family>
           <pointsize>12</pointsize>
           <bold>true</bold>
          </font>
         </property>
         <property name="text">
          <string>Điểm</string>
         </property>
        </widget>
       </widget>
      </item>
      <item>
       <widget class="QGroupBox" name="groupBox_5">
        <property name="styleSheet">
         <string notr="true">border: None;</string>
        </property>
        <property name="title">
         <string/>
        </property>
        <widget class="QPushButton" name="them">
         <property name="geometry">
          <rect>
           <x>20</x>
           <y>70</y>
           <width>161</width>
           <height>41</height>
          </rect>
         </property>
         <property name="font">
          <font>
           <family>Tahoma</family>
           <pointsize>12</pointsize>
           <bold>true</bold>
          </font>
         </property>
         <property name="styleSheet">
          <string notr="true">border: 3px solid rgb(0,0,0);
background-color: rgb(85, 170, 255);</string>
         </property>
         <property name="text">
          <string>Thêm</string>
         </property>
         <property name="icon">
          <iconset>
           <normalon>:/images/6_n.png</normalon>
          </iconset>
         </property>
        </widget>
        <widget class="QPushButton" name="thoat">
         <property name="geometry">
          <rect>
           <x>20</x>
           <y>120</y>
           <width>161</width>
           <height>41</height>
          </rect>
         </property>
         <property name="font">
          <font>
           <family>Tahoma</family>
           <pointsize>12</pointsize>
           <bold>true</bold>
          </font>
         </property>
         <property name="styleSheet">
          <string notr="true">border: 3px solid rgb(0,0,0);
background-color: rgb(85, 170, 255);</string>
         </property>
         <property name="text">
          <string>Thoát</string>
         </property>
         <property name="icon">
          <iconset>
           <normalon>:/images/32_n.png</normalon>
          </iconset>
         </property>
        </widget>
        <widget class="QPushButton" name="tim_kiem">
         <property name="geometry">
          <rect>
           <x>20</x>
           <y>20</y>
           <width>161</width>
           <height>41</height>
          </rect>
         </property>
         <property name="font">
          <font>
           <family>Tahoma</family>
           <pointsize>12</pointsize>
           <bold>true</bold>
          </font>
         </property>
         <property name="styleSheet">
          <string notr="true">border: 3px solid rgb(0,0,0);
background-color: rgb(85, 170, 255);</string>
         </property>
         <property name="text">
          <string>Tìm kiếm</string>
         </property>
         <property name="icon">
          <iconset>
           <normalon>:/images/893_n.png</normalon>
          </iconset>
         </property>
        </widget>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item row="0" column="1">
    <widget class="QGroupBox" name="groupBox_2">
     <property name="styleSheet">
      <string notr="true">border: 3px solid rgb(0, 112, 192);
border-radius: 10px;
background-color:rgb(0, 112, 192);
</string>
     </property>
     <property name="title">
      <string/>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout_2">
      <item>
       <widget class="QTableWidget" name="danh_sach_cau_hoi">
        <property name="font">
         <font>
          <family>Tahoma</family>
          <pointsize>10</pointsize>
         </font>
        </property>
        <property name="styleSheet">
         <string notr="true">background-color: rgb(255, 255, 255);
border: None;
border-radius: 0px;
</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="../../resource.qrc"/>
 </resources>
 <connections/>
</ui>
